[2021-12-05 21:18:52,296] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2021-12-06T02:13:13.036416+00:00 [queued]>
[2021-12-05 21:18:52,307] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2021-12-06T02:13:13.036416+00:00 [queued]>
[2021-12-05 21:18:52,307] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 21:18:52,307] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-05 21:18:52,307] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 21:18:52,316] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): run_after_loop> on 2021-12-06 02:13:13.036416+00:00
[2021-12-05 21:18:52,319] {standard_task_runner.py:52} INFO - Started process 4843 to run task
[2021-12-05 21:18:52,324] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2021-12-06T02:13:13.036416+00:00', '--job-id', '14', '--raw', '--subdir', '/Users/ballakeerthi/opt/anaconda3/envs/airflow/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T/tmp5tyfixq7', '--error-file', '/var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T/tmp74f5unfu']
[2021-12-05 21:18:52,326] {standard_task_runner.py:77} INFO - Job 14: Subtask run_after_loop
[2021-12-05 21:18:57,357] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2021-12-06T02:13:13.036416+00:00 [running]> on host zipcodes-MacBook-Pro-3.local
[2021-12-05 21:19:02,384] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2021-12-06T02:13:13.036416+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-06T02:13:13.036416+00:00
[2021-12-05 21:19:02,385] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T
[2021-12-05 21:19:02,386] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2021-12-05 21:19:02,391] {subprocess.py:85} INFO - Output:
[2021-12-05 21:19:02,396] {subprocess.py:89} INFO - 1
[2021-12-05 21:19:02,397] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-05 21:19:02,410] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20211206T021313, start_date=20211206T021852, end_date=20211206T021902
[2021-12-05 21:19:02,455] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-05 21:19:02,470] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
